python/paddle/fluid/tests/unittests/dygraph_to_static/test_mnist.py

```diff
@@ -106,9 +106,13 @@ def forward(self, inputs, label=None):
             acc = fluid.layers.accuracy(input=x, label=label)
             loss = fluid.layers.cross_entropy(x, label)
             avg_loss = fluid.layers.mean(loss)
-            return x, acc, avg_loss
-        else:
-            return x
+
+        # TODO: Uncomment code after "return" statement can be transformed correctly.
```

Comment:
Why is this return commented? Convert_if shouldn't transform because `if label is not None` is not tensor.

Reply:
If not commented , this `forward` function is transformed as follows, but it run python if actually because the pred `label is not None` is not a Tensor.  `Return` statements needs to be transformed specially. I will do this in the future PR.
```
def forward(self, inputs, label=None):
    x = fluid.dygraph.dygraph_to_static.convert_call(self.inference)(inputs)

    def true_fn_0(label, x):
        acc = fluid.layers.accuracy(input=x, label=label)
        loss = fluid.layers.cross_entropy(x, label)
        avg_loss = fluid.layers.mean(loss)
        return x, acc, avg_loss
        return

    def false_fn_0(x):
        return x
        return
    fluid.dygraph.dygraph_to_static.convert_operators.convert_ifelse(label
         is not None, lambda : fluid.dygraph.dygraph_to_static.convert_call
        (true_fn_0)(label, x), lambda : fluid.dygraph.dygraph_to_static.
        convert_call(false_fn_0)(x))
```

Reply:
Discussed offline, we will add it back after we support early return.